### ISSUE TITLE
st_write: values not successfully written

### ISSUE THREAD

**Issue by tghoward** [HeritageNetwork/Regional_SDM#69]
It looks like I am running into this problem a few times throughout the script. The first is at the bottom of script 2, at `st_write`
```
> st_write(points_attributed, paste0("model_input/", filename), delete_layer = T)
Writing layer `bombferv_20181220_140139_att' to data source `model_input/bombferv_20181220_140139_att.shp' using driver `ESRI Shapefile'
features:       1271
fields:         51
geometry type:  Point
There were 12 warnings (use warnings() to see them)
> warnings()[1:2]
Warning messages:
1: In CPL_write_ogr(obj, dsn, layer, driver, as.character(dataset_options),  ... :
  GDAL Message 1: Value -13706267 of field crvprox100 of feature 533 not successfully written. Possibly due to too larger number with respect to field width
2: In CPL_write_ogr(obj, dsn, layer, driver, as.character(dataset_options),  ... :
  GDAL Message 1: Value -12236988 of field crvslpx100 of feature 540 not successfully written. Possibly due to too larger number with respect to field width
```

**Comment by tghoward** [comment]
I'm not sure if/how we can define field widths, or even if that's a red herring.

**Comment by tghoward** [comment]
hmm, The data *are* in the output shapefile. That doesn't make sense there's a warning.

**Comment by dnbucklin** [comment]
Is that really a valid value for that variable?!

Seems the value wouldn't be a problem for a Double field type in a shapefile. Maybe the warning is a bug in the underlying function.

**Comment by tghoward** [comment]
yes, I think it might be a bug in the underlying function. I can replicate with a very simple script and will post on `sf` site. I think these values are `curvature * 1000` which might be why they can get so large (and variable). I think the variability might be what's wrong.

**Comment by tghoward** [comment]
Found it. Already a closed issue:  https://github.com/r-spatial/sf/issues/306

In the shapefile driver and so can't be changed by sf. Suggestion is to not shapefiles!

**Comment by dnbucklin** [comment]
Ha, interesting...

I guess we could `suppressWarnings` in that `st_write` call. Alternatively, does that file need to be a shapefile? Could just write it to a CSV; we do that with aquatic.

**Comment by tghoward** [comment]
or start using geopackages?  I don't have any idea whether that would be faster or slower or the same.

**Comment by dnbucklin** [comment]
Might be a good idea for the spatial stuff, though I haven't used them enough to know. 

But this file doesn't need to be spatial. Same with the `_bkg_clean.shp` file.
